src/signature/policy/policy_requirement.rs

```diff
@@ -7,17 +7,14 @@ use anyhow::{anyhow, Result};
 use oci_distribution::secrets::RegistryAuth;
 use serde::*;
 
-#[cfg(feature = "cosign")]
-use crate::signature::{
-    image::Image,
-    mechanism::{cosign::CosignParameters, simple::SimpleParameters, SignScheme},
-};
-
-#[cfg(not(feature = "cosign"))]
-use crate::signature::{
-    image::Image,
-    mechanism::{simple::SimpleParameters, SignScheme},
-};
+use crate::signature::image::Image;
+#[cfg(feature = "signature-cosign")]
+use crate::signature::mechanism::cosign::CosignParameters;
+#[cfg(feature = "signature-simple")]
+use crate::signature::mechanism::simple::SimpleParameters;
+use crate::signature::mechanism::SignScheme;
+#[cfg(all(feature = "signature-cosign", feature = "signature-simple"))]
+compile_error!("both signature-cosign and signature-simple are enabled");
```

Comment:
We surely allow both of the features are enabled, although currently some of the scenarios only use one of them.

Reply:
BTW, should we make feature `keywrap-native` and `keywrap-grpc` mutually exclusive since it may used by different runtime like `kata-cc` and `enclave-cc`?

Reply:
Fixed